easybuild/tools/docs.py

```diff
@@ -707,6 +708,44 @@ def list_software_txt(software, detailed=False):
     return '\n'.join(lines)
 
 
+def gather_reverse_dependencies():
+    """
+    Gather reverse dependencies from installed modules.
+
+    :return: dictionary with reverse dependencies
+    """
+    reverse_dependencies = {}
+
+    module_avail_cmd = "module -t avail 2>&1 | grep -v ':$'"
+    module_purge_load_list_cmd = "module purge; module load %s; module -t list 2>&1 | grep -v %s"
```

Comment:
Using `purge` is troublesome on some systems, and fairly expensive with Lmod...

How about just running this in a subshell, since we only care about the output, not the changes to the environment?

Or use `ml show` (which only gives direct deps though), that would also make it significantly faster...